integration_tests/tests/e2e-tests.ts

```diff
@@ -103,11 +109,16 @@ describe("\n Upload pdf invoice to raw invoice bucket and generate transactions
   `(
     "results retrieved from BillingAndTransactionsCuratedView should match with expected $testCase,$eventTime,$transactionQty,$billingQty",
     async (data) => {
-      await generateTransactionEventsViaFilterLambda(
-        data.eventTime,
-        data.transactionQty,
-        eventName
-      );
+      for (let i = 0; i < data.transactionQty; i++) {
+        const createEventPayload = await generateTestEvent({
```

Comment:
Similar to my previous comment, I think the variable name can just be eventPayload. 
```suggestion
        const eventPayload = await generateTestEvent({
```

Reply:
done